pennylane/ops/qubit/hamiltonian.py

```diff
@@ -216,6 +218,9 @@ def __init__(
         # while H.coeffs is the original tensor
         super().__init__(*coeffs_flat, wires=self._wires, id=id, do_queue=do_queue)
 
+    def _check_batching(self, params):
+        """Override for Hamiltonian, batching is not yet supported."""
```

Comment:
```suggestion
        """Override for Hamiltonian, batching is not yet supported."""
        pass
```

Reply:
I thought so too, but pylint said the `pass` was unnecessary!